src/d_main.c

```diff
@@ -554,6 +542,9 @@ void D_DoAdvanceDemo(void)
     int i = W_CheckNumForName("TITLEPIC");
     int j = W_CheckNumForName("DMENUPIC");
 
+    if (!strncasecmp("DOOMPRES", W_WadNameForLump(i), 8))
```

Comment:
Is this because `DOOMPRES.WAD` does not have a `TITLEPIC` lump, or because it is unused? The former should already be covered by the existing logic.

Reply:
`TITLEPIC` is in a different "sneas" format. The beta starts right on E1M2, with no menu or title. Also it's misleading:
![image](https://user-images.githubusercontent.com/5077629/182369719-67f51703-918d-451b-a666-c6498e2dea77.png)